src/components/CreateElement/createElement.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CreateElementComponent = React.createElement(
```

Comment:
here it's not a **component** but **element**. Please, create a component which returns an element created with `React.createElement` here, and remove the workaround with `componentDidMount`

Reply:
Fixed, please review the last commit.